broker-cli/broker-daemon-client/index.js

```diff
@@ -73,8 +74,12 @@ class BrokerDaemonClient {
       if (!this.username) throw new Error('No username is specified for authentication')
       if (!this.password) throw new Error('No password is specified for authentication')
 
-      // Go back to the ./broker-cli/certs directory from the current directory
-      this.cert = readFileSync(path.join(PROJECT_ROOT, this.certPath))
+      let certPathParts = this.certPath.split(path.sep)
+      if (certPathParts[0] === '~') {
+        certPathParts[0] = os.homedir()
+      }
+      const certPath = path.join(...certPathParts)
```

Comment:
[nit] I dont think we need to spread here, since path.join should take the array

Reply:
this is so wrong ^^ it takes segments, not an array. sorry @martineehrlich